.go-version

```diff
@@ -1 +1 @@
-1.21.13
+1.22.0
```

Comment:
The `.go-version` from Beats and Elastic-Agent is `1.22.8`, so I suggest using the same here.
```suggestion
1.22.8
```